root-feature/Navigation/AuthNavigation.tsx

```diff
@@ -0,0 +1,75 @@
+import { TiFAPI } from "@api-client/TiFAPI"
+import { createAWSTiFAPIFetch } from "@api-client/aws"
+import {
+  ForgotPasswordParamsList,
+  createForgotPasswordScreens
+} from "@auth/forgot-password"
+import { createForgotPasswordEnvironment } from "@auth/forgot-password/Environment"
+import {
+  CognitoSignInAuthenticator,
+  SignInParamsList,
+  createSignInScreens
+} from "@auth/sign-in"
+import {
+  SignUpParamsList,
+  cognitoConfirmSignUpWithAutoSignIn,
+  createSignUpEnvironment,
+  createSignUpScreens
+} from "@auth/sign-up"
+import { Auth } from "@aws-amplify/auth"
+import { BASE_HEADER_SCREEN_OPTIONS } from "@components/Navigation"
+import { createStackNavigator } from "@react-navigation/stack"
+import React from "react"
+
+export type AuthNavigationStackParamList = SignInParamsList &
+  SignUpParamsList &
+  ForgotPasswordParamsList
+
+const Stack = createStackNavigator<AuthNavigationStackParamList>()
+
+const tiFAPI = new TiFAPI(
+  createAWSTiFAPIFetch(
+    new URL("https://623qsegfb9.execute-api.us-west-2.amazonaws.com/prod/")
+  )
+)
```

Comment:
We could add a static property for this so one could just use `TiFAPI.production` or something like that. Also, I think there's an environment variable for the production URL called `API_URL` which you can access via:

```ts
import { API_URL } from "env"
```